test/OrleansTestKit.Tests/Tests/GrainProbeTests.cs

```diff
@@ -143,8 +146,9 @@ public async Task FactoryProbe()
             pong.Verify(p => p.Pong(), Times.Once);
         }
 
+        #region FactoryProbeWithMultipleNewProbes
```

Comment:
Please remove the regions. This isn't aligned with the current code base's style.

As an aside... I should get `dotnet-format` added to the GitHub Actions pipeline to help flag & address these small things more quickly.

Reply:
of course, will remove

Reply:
done